vscode-extension/src/languageClient.ts

```diff
@@ -33,7 +34,10 @@ export function createAndStartLanguageClient(context: RelayExtensionContext) {
     options: {
       cwd: context.relayBinaryExecutionOptions.rootPath,
     },
-    command: context.relayBinaryExecutionOptions.binaryPath,
+    command: path.resolve(
+      context.relayBinaryExecutionOptions.rootPath,
+      context.relayBinaryExecutionOptions.binaryPath,
+    ),
```

Comment:
@leoasis doesn't this break down if you specify an absolute path to a binary in your config? 

the use case for the absolute path is when I'm testing a locally built binary

Reply:
According to https://nodejs.org/api/path.html#pathresolvepaths it shouldn't:

_The given sequence of paths is processed from right to left, with each subsequent path prepended until an absolute path is constructed. For instance, given the sequence of path segments: /foo, /bar, baz, calling path.resolve('/foo', '/bar', 'baz') would return /bar/baz because 'baz' is not an absolute path but '/bar' + '/' + 'baz' is._

In the case you mentioned, the binary path would already be absolute so it should return it as is.

Haven't tried this though, will confirm on Monday or you can give it a go if you have a way to try this out before then

Reply:
Ok I just tried this out locally with an absolute path and as expected it continues to work.